### ISSUE TITLE
Modern Taxes Archetype definition

### ISSUE THREAD

**Issue by Never28** [Badaro/MTGOFormatData#51]
With Bowmaster in the meta, there are some lists that cuts Thalia for other two drops (especially the orzhov ones that plays Bowmaster, Arbiter and SFM). On the other hand, Archon has become a core piece of the deck

For a better detection I would suggest to change the condition
`{
      "Type": "InMainboard",
      "Cards": ["Thalia, Guardian of Thraben"]
    },`

Into
`{
      "Type": "TwoOrMoreInMainOrSideboard",
      "Cards": ["Thalia, Guardian of Thraben", "Leonin Arbiter", "Archon of Emeria"]
    },`

**Comment by Badaro** [comment]
Ran a diff with this change.

```
* modern-league-2019-12-17 -> kiko changed from Esper Taxes to Esper Blink
* modern-league-2019-12-20 -> Gods_Shadow changed from Mono White Taxes to Mono White Blink
* modern-challenge-2019-12-2212049241 -> kiko changed from Esper Taxes to Esper Blink
* modern-preliminary-2020-01-0112052938 -> WangoTango73 changed from Azorius Taxes to Azorius Blink
* modern-league-2020-01-21 -> Heibing changed from Mono White Taxes to Mono White Midrange
* modern-league-2020-02-11 -> yriel changed from Mono White Eldrazi Taxes to Mono White Midrange
* modern-league-2020-03-13 -> fabee1 changed from Azorius Taxes to Azorius Midrange
* modern-players-tour-qualifier-2020-03-1512106036 -> kponceMTG changed from Mono White Taxes to Mono White Midrange
* modern-preliminary-2020-03-1712106076 -> KelMasterP changed from Mono White Taxes to Mono White Midrange
* modern-preliminary-2020-03-1712106076 -> kponceMTG changed from Mono White Taxes to Mono White Midrange
* modern-preliminary-2020-03-1712106076 -> lukystrike changed from Azorius Taxes to Azorius Midrange
* modern-league-2020-03-20 -> yriel changed from Mono White Eldrazi Taxes to Mono White Midrange
* modern-preliminary-2020-03-2012110933 -> lukystrike changed from Azorius Taxes to Azorius Midrange
* modern-league-2020-04-28 -> 6argamel changed from Abzan Taxes to Abzan Midrange
* modern-league-2020-05-26 -> AnTiRayMan changed from Selesnya Taxes to Selesnya Midrange
* modern-challenge-2020-08-2312198548 -> AnTiRayMan changed from Selesnya Taxes to Selesnya Midrange
* modern-challenge-2020-09-2612211158 -> kiko changed from Mono White Taxes to Mono White Midrange
* modern-league-2020-09-29 -> thelastgod changed from Selesnya Midrange to Selesnya Taxes
* modern-league-2020-10-06 -> seasonofmists changed from Selesnya Blink to Selesnya Eldrazi Taxes
* modern-league-2020-10-06 -> Gods_Shadow changed from Mono White Taxes to Mono White Midrange
* the-dive-down-nation-modern-fnm-1024-edition-3865-2020-10-24 -> Samuel Wallace changed from Azorius Midrange to Azorius Taxes
* modern-challenge-2020-11-2912233116 -> random_deck changed from Selesnya Midrange to Selesnya Taxes
* modern-preliminary-2021-01-2912256309 -> kiko changed from Azorius Taxes to Azorius Blink
* modern-challenge-2021-04-1812285852 -> kiko changed from Orzhov Taxes to Orzhov Midrange
* modern-challenge-2021-06-0512299469 -> kiko changed from Azorius Taxes to Azorius Blink
* modern-league-2021-06-08 -> VerMahilen changed from Boros Taxes to Boros Midrange
* modern-league-2021-06-08 -> SpiderSpace changed from WUBR Taxes to WUBR Blink
* modern-league-2021-06-11 -> _Neptune changed from Azorius Taxes to Azorius Blink
* modern-league-2021-06-11 -> VerMahilen changed from Boros Taxes to Boros Midrange
* modern-challenge-2021-06-2612309177 -> kiko changed from Azorius Taxes to Azorius Blink
* modern-league-2021-07-02 -> Gionax changed from Orzhov Taxes to Orzhov Blink
* modern-challenge-2021-07-0412312091 -> kiko changed from Azorius Taxes to Azorius Blink
* modern-league-2021-07-06 -> AnnndWhammy changed from Boros Taxes to Boros Midrange
* modern-league-2021-07-16 -> RespectTheCat changed from Orzhov Taxes to Unknown
* cmc-modern-cup-6-6991-2021-07-17 -> Fucearete11 changed from Mono White Taxes to Unknown
* modern-league-2021-07-20 -> RespectTheCat changed from Orzhov Taxes to Unknown
* modern-league-2021-07-27 -> DannBjorn changed from Azorius Taxes to Azorius Midrange
* manatraders-series-modern-july-2021-2021-07-31 -> RespectTheCat changed from Orzhov Taxes to Unknown
* manatraders-series-modern-july-2021-2021-07-31 -> klien7 changed from Orzhov Taxes to Unknown
* modern-league-2021-10-05 -> deathandcatmix changed from Mono White Taxes to Mono White Midrange
* modern-league-2021-10-12 -> Never28 changed from Mono White Taxes to Mono White Midrange
* modern-preliminary-2021-11-0212351502 -> BantQuirionPleaseWrk changed from Abzan Midrange to Abzan Taxes
* clasificatorio-al-antes-conocido-como-nacional-modern-8138-2021-11-07 -> Aldo Borghes changed from Mono White Blink to Mono White Taxes
* 9o-torneio-4a-liga-arena-guardians-modern-8340-2021-11-15 -> Bruno Apolonio Rodrigues changed from Mono White Taxes to Mono White Blink
* modern-league-2021-11-16 -> Purplegambit changed from Selesnya Taxes to Selesnya Midrange
* modern-league-2021-12-10 -> Bolas299 changed from Bant Taxes to Bant Midrange
* modern-league-2021-12-28 -> NorthLagos changed from Selesnya Taxes to Selesnya Midrange
* modern-league-2022-01-04 -> Chipkay changed from Mono White Taxes to Mono White Midrange
* modern-league-2022-01-07 -> Chipkay changed from Mono White Taxes to Mono White Midrange
* modern-league-2022-01-18 -> Parrit changed from Boros Taxes to Boros Blink
* modern-league-2022-01-25 -> 1654706743 changed from Esper Taxes to Esper Midrange
* modern-league-2022-03-08 -> Offbeat_Hero changed from Azorius Taxes to Azorius Blink
* nrg-series-20000-showdown-fort-wayne-modern-8591-2022-04-02 -> Robert Boyer changed from Selesnya Taxes to Selesnya Midrange
* modern-league-2022-04-05 -> Pablo Figuero changed from 5 Color Taxes to 5 Color Midrange
* scg-con-dallas-friday-1230pm-modern-trial-9199-2022-04-08 -> Aaron OHanlon changed from Mono White Taxes to Mono White Blink
* scg-con-dallas-friday-230pm-modern-trial-9200-2022-04-08 -> Remo Khuri changed from Azorius Eldrazi Taxes to Azorius Midrange
* scg-con-dallas-modern-30k-9196-2022-04-09 -> Aaron OHanlon changed from Mono White Taxes to Mono White Blink
* scg-con-dallas-modern-30k-9196-2022-04-09 -> Remo Khuri changed from Azorius Eldrazi Taxes to Azorius Midrange
* scg-con-dallas-modern-10k-9197-2022-04-10 -> Remo Khuri changed from Azorius Eldrazi Taxes to Azorius Midrange
* modern-league-2022-05-20 -> Tartalo changed from Mono White Taxes to Mono White Midrange
* nrg-series-5000-trial-minneapolis-modern-9603-2022-05-22 -> Jon Puffer changed from Selesnya Taxes to Selesnya Midrange
* 5o-torneio-6a-liga-arena-guardians-modern-10582-2022-06-13 -> Romulo Bernardino changed from Jeskai Taxes to Jeskai Blink
* modern-league-2022-06-28 -> volpert changed from Boros Midrange to Boros Taxes
* manatraders-series-modern-june-2022-2022-06-30 -> Offbeat_Hero changed from Azorius Taxes to Azorius Blink
* modern-league-2022-07-01 -> volpert changed from Boros Midrange to Boros Taxes
* 8o-torneio-6a-liga-arena-guardians-modern-10998-2022-07-04 -> Igor Macedo changed from Mono White Taxes to Mono White Midrange
* modern-trial-scg-con-syracuse-friday-100-pm-10152-2022-07-08 -> Jason Lu changed from Orzhov Taxes to Orzhov Midrange
* modern-trial-scg-con-syracuse-friday-400-pm-10153-2022-07-08 -> Jason Lu changed from Orzhov Taxes to Orzhov Midrange
* modern-30k-scg-con-syracuse-saturday-1000-am-10155-2022-07-09 -> Jason Lu changed from Orzhov Taxes to Unknown
* modern-5k-scg-con-syracuse-sunday-900-am-10156-2022-07-10 -> Jason Lu changed from Orzhov Taxes to Unknown
* modern-league-2022-07-22 -> kponceMTG changed from Mono White Taxes to Mono White Midrange
* modern-league-2022-07-29 -> Tyuhan changed from Boros Taxes to Boros Blink
* modern-league-2022-08-30 -> Offbeat_Hero changed from Azorius Taxes to Unknown
* modern-league-2022-09-30 -> YoungProdigy changed from Mono White Midrange to Mono White Taxes
* 6o-torneio-7a-liga-arena-guardians-modern-12289-2022-10-10 -> Nicollas Alves da Silva Diniz changed from Mono White Taxes to Mono White Midrange
* 7o-torneio-7a-liga-arena-guardians-modern-12384-2022-10-18 -> Nicollas Alves da Silva Diniz changed from Mono White Taxes to Mono White Midrange
* modern-league-2023-01-31 -> Drakus77 changed from Selesnya Taxes to Selesnya Midrange
* modern-league-2023-02-03 -> Drakus77 changed from Selesnya Taxes to Selesnya Midrange
* special-20-entry-20k-rcq-top-4-modern-scg-con-charlotte-saturday-1000-am-14035-2023-03-04 -> Stephen Fowler changed from Mono White Taxes to Unknown
* 2023-hunter-burton-memorial-open-modern-13072-2023-03-18 -> Lawson Zandi changed from Boros Taxes to Boros Midrange
* grand-open-qualifier-prague-modern-14914-2023-04-01 -> MariusK changed from Mono White Blink to Mono White Taxes
* modern-challenge-2023-05-2712552641 -> Traumatismes changed from Conflict(Mardu Taxes,Winota) to Winota
* grand-open-qualifier-bologna-modern-16657-2023-07-08 -> Igor Makarev changed from WBRG Taxes to WBRG Midrange
* modern-league-2023-07-21 -> Zeus87 changed from Azorius Midrange to Azorius Taxes
* modern-super-qualifier-2023-07-3012570116 -> Zeus87 changed from Azorius Midrange to Azorius Taxes
* modern-league-2023-08-01 -> CharonsObol changed from Mono White Midrange to Mono White Taxes
* modern-league-2023-08-04 -> The_Catfin changed from Mono White Blink to Mono White Taxes
* modern-league-2023-08-08 -> CharonsObol changed from Mono White Midrange to Mono White Taxes
* modern-league-2023-08-18 -> CharonsObol changed from Mono White Midrange to Mono White Taxes
* doghouse-vi-lord-of-the-bings-modern-17364-2023-08-19 -> ImRadon changed from Conflict(Grief Blade,Orzhov Taxes) to Grief Blade
* 20k-rcq-top-4-modern-scg-con-columbus-saturday-1000-am-16954-2023-09-02 -> CorbanD changed from Mono White Midrange to Mono White Taxes
* modern-league-2023-09-05 -> nex7 changed from Orzhov Midrange to Orzhov Taxes
* modern-league-2023-09-05 -> Zeus87 changed from Mono White Blink to Mono White Taxes
* modern-league-2023-09-15 -> _DG_ changed from Mono White Midrange to Mono White Taxes
* modern-league-2023-10-17 -> Aldo_80 changed from Mono White Midrange to Mono White Taxes
* grand-open-qualifier-sofia-modern-32882-2023-10-21 -> Leonardo Tozzi changed from Mono White Midrange to Mono White Taxes
* modern-league-2023-10-27 -> Mr-Jackpots changed from Mono White Midrange to Mono White Taxes
* manatraders-series-modern-october-2023-2023-10-31 -> Aldo_80 changed from Mono White Midrange to Mono White Taxes
* manatraders-series-modern-october-2023-2023-10-31 -> Devantoi changed from Mono White Midrange to Mono White Taxes
* eternal-weekend-europe-2023-modern-mainevent-38334-2023-11-19 -> Björn Fälber changed from Mono White Midrange to Mono White Taxes
* modern-league-2023-11-24 -> _DG_ changed from Orzhov Midrange to Orzhov Taxes
* nrg-series-10000-showdown-madison-wisconsin-modern-15654-2023-11-25 -> Alex Reynolds changed from Mono White Midrange to Mono White Taxes
* modern-league-2023-12-08 -> CharonsObol changed from Orzhov Midrange to Orzhov Taxes
* modern-league-2023-12-08 -> OrzhovControl changed from Mono White Midrange to Mono White Taxes
* grand-open-qualifier-barcelona-2023-modern-38286-2023-12-09 -> Sadie Braun changed from Orzhov Midrange to Orzhov Taxes
```

I was a bit concerned at first since lot of older lists were being reclassified to a more generic "Blink" or "Midrange" archetypes, but I did a sampling of the non-League ones and they _not_ traditional Taxes lists, so the new classification actually looked better.

Just to give some examples, this one is now classified as Blink:
![image](https://github.com/Badaro/MTGOFormatData/assets/807722/27895bf4-ab64-4e35-b8f6-3343b31f3386)

And this one is now classified as Midrange.
![image](https://github.com/Badaro/MTGOFormatData/assets/807722/9f507313-f031-46e3-bd34-7fc942c01e7a)

Both changes seem positive to me. Also, this change also doesn't create any conflicts (which is the usualy red flag when modifying archetypes).

**_TL;DR: This looks like good change, I'm commiting and pushing it now. :)_**

**Comment by Never28** [comment]
Yeah, having those labeled as Blink is not ideal, they are prior to the Archon print. Sadly even on the taxes community is difficult to say what is taces and what is not (is monow human with Esper Sentinel, Thalia, Anointed Peacekeeper a taxes deck? Some would say yes even if it doesn't include the SFM package). Probably you could add Flickerwisp to the TwoOrMoreInMainOrSideboard, it was rtpically a taxes card but now doesn't see as much play, it should cover also older taxes lists. You could consider to add SFM to the condition instead depending if you want to label a stoneless build under Monow Human or not.
